pkg/config/config.go

```diff
@@ -145,6 +145,11 @@ func init() {
 	// Kube ApiServer
 	Datadog.SetDefault("kubernetes_kubeconfig_path", "")
 
+	// Datadog cluster agent
+	Datadog.SetDefault("cluster_agent_auth_token", "")
+	Datadog.SetDefault("cluster_agent_url", "")
+	Datadog.SetDefault("cluster_agent_kubernetes_service_name", "dca")
```

Comment:
maybe `cluster_agent` options are a good candidate to be nested options, e.g.:
```yaml
cluster_agent:
  auth_token: aaaaaaaaaa
  url: https://foo:12345
```

and the related values can be set/accessed with, for example: `Datadog.SetDefault("cluster_agent.auth_token", "")`, `Datadog.Get("cluster_agent.auth_token", "")`

Reply:
Indeed, that's better ! :+1: